.github/actions/start-aws-runner/action.yml

```diff
@@ -41,7 +41,7 @@ runs:
         aws-region: us-east-2
     - name: Start EC2 runner
       id: start-ec2-runner
-      uses: supertopher/ec2-github-runner@v15.0.5
+      uses: supertopher/ec2-github-runner@base64v1.0.5
```

Comment:
I know I'm not exactly semantic here, but adding `base64` feels like a better clue for future readers